cmd/gitbase/command/server.go

```diff
@@ -227,7 +228,7 @@ func (c *Server) buildDatabase() error {
 
 	c.sharedCache = cache.NewObjectLRU(c.CacheSize * cache.MiByte)
 
-	c.rootLibrary = libraries.New(libraries.Options{})
+	c.rootLibrary = libraries.New(nil)
```

Comment:
I would try to get rid of this mysterious `nil`
how about sth. like `context interface`:
```go
lib := library.New().WithOption(...).WithSiva(...)
//or
lib = library.WithOption(library.New())
lib = library.WithSiva(lib, sivaOpts)
```
WDYT?